### PR TITLE
Update registry.json

### PR DESCRIPTION
**Pull request by chongsoon**
To add NTU's Ropsten Store in registry.json.

### CODE REVIEW

**Comment by rjchow** [comment]
where are your templates?

**Comment by chongsoon** [comment]
We are currently still in the development of the templates. Would we be able to add to the registry without the templates at the moment? Or should we upload with the template together?

**Comment by rjchow** [comment]
yup do it together with the templates, its fine if you're not ready to merge -- you can just use the PR preview feature to do your user testing

**Comment by chongsoon** [comment]
I just committed a new certificate template.

**Comment by chongsoon** [comment]
I am unable to verify certificate with the latest commit's PR preview. I was given an error message stating "Certificate from unregistered institution". Please advise.

**Comment by sumitnitsurat** [comment]
Please make sure you are using the published certificate on the same network and that store is in registry.json file. Also please check your test are not passing in sync with master.

File: src/components/CertificateTemplates/tlds/sg/edu/ntu/index.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import dynamic from "next/dynamic";
+import { addDirToTemplatePath } from "template-utils/addDirToTemplatePath";
+
+const templates = {
+  "2019-ntu-degree": dynamic(() =>
```
Comment (sumitnitsurat) [inline src/components/CertificateTemplates/tlds/sg/edu/ntu/index.js]: please check the test for it.  return the dynamic function to the key like this - "2019-ntu-degree":  () => dynamic(() =>"  or follow the standard everyone followed to keep the dynamic and template const separated.

File: src/components/CertificateTemplates/tlds/sg/edu/ntu/2019-July-NTU-UG/ntu_cert_1.opencert [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    "id": "03699657-3515-45b1-81ae-020526a7a7dd:string:SERIAL-2018-08-01-001",
+    "$template": "ede56167-5d82-42de-abd4-99246642607d:string:sg/edu/ntu/2019-ntu-degree",
+    "issuedOn": "3f082cf0-b915-4770-b097-b4f9948864c3:string:2018-08-01T00:00:00+08:00",
+    "name": "c4866b88-bcc5-4c64-abb3-49d58f5e0844:string:NTU Degree in EEE",
+    "admissionDate": "6c63029a-ad46-46f5-b2f6-6f10d6736c1c:string:2016-08-01T00:00:00+08:00",
+    "graduationDate": "a8857643-6255-4162-94bc-868151bab629:string:2018-08-01T00:00:00+08:00",
+    "issuers": [
+      {
+        "name": "926526e4-783a-46d8-87a0-38116d695005:string:Pace College",
+        "did": "38f0dc3e-7b49-4220-acea-f15ee28d22f4:string:DID:SG-UEN:PACE123456",
```
Comment (sumitnitsurat) [inline src/components/CertificateTemplates/tlds/sg/edu/ntu/2019-July-NTU-UG/ntu_cert_1.opencert]: Please don`t use "did" as a key. Use "nric" or "fin" as a key instead.

File: static/registry.json [hunk 3]
```diff
@@ -12,6 +12,7 @@
 		},
 		"Ropsten": {
 			"0x5f007251c78cA92d3053492cCf0EDC161063624c": "ROPSTEN: Nanyang Polytechnic",
+      "0xe7F411Bf96BA3F9B9afd639C90eF921574b7d37D": "ROPSTEN: Nanyang Technological University",
```
Comment (sumitnitsurat) [inline static/registry.json]: Duplicate address.

File: src/components/CertificateTemplates/tlds/sg/edu/ntu/2019-July-NTU-UG/integration.spec.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    Promise.resolve()
+  );
+
+test("NTU Undergrad certificate is rendered correctly", async t => {
+  // Uploads certificate via dropzone
+  await t.setFilesToUpload("input[type=file]", [Certificate]);
+
+  // Certificate tabs rendered
+  await t.expect(TemplateTabList.textContent).contains("Certificate");
+  await t.expect(TemplateTabList.textContent).contains("Transcript");
```
Comment (sumitnitsurat) [inline src/components/CertificateTemplates/tlds/sg/edu/ntu/2019-July-NTU-UG/integration.spec.js]: you haven`t use Transcript in the certificate due to which your integration test is getting failed. Please fix that.

File: src/components/CertificateTemplates/tlds/sg/edu/ntu/2019-July-NTU-UG/certificate_backup.js [hunk 5]
```diff
@@ -0,0 +1,69 @@
+import PropTypes from "prop-types";
```
Comment (sumitnitsurat) [inline src/components/CertificateTemplates/tlds/sg/edu/ntu/2019-July-NTU-UG/certificate_backup.js]: Is this file used any where or you are going to use?

Comment (chongsoon) [reply src/components/CertificateTemplates/tlds/sg/edu/ntu/2019-July-NTU-UG/certificate_backup.js]: i have removed the file in the latest commit.

**Comment by sumitnitsurat** [comment]
Please fix the lint issues also. Run "npm run lint:fix" and "npm run test" before committing to check if everything fine with your commit.

**Comment by chongsoon** [comment]
hey, just did a commit and tried the preview link, the viewer stated that my certificate is invalid. it is ok when I tested locally. please advise. thanks

**Comment by sumitnitsurat** [comment]
Hi..Can you please make sure you are testing development or production preview.   "https://deploy-preview-254--opencerts-development.netlify.com/" if you are testing a test certificate and "https://deploy-preview-254--opencerts-production.netlify.com/" if you are testing for prod cert.

**Comment by chongsoon** [comment]
oh noted, i had been using the netlify preview link. thanks a lot.

**Comment by sumitnitsurat** [comment]
please request for re-review when you are done with the suggested changes.

**Pull request closed without merging**